### ISSUE TITLE
Importing dynamic-graph python modules

### ISSUE THREAD

**Issue by florent-lamiraux** [stack-of-tasks/dynamic-graph-python#54]
When importing python modules, some libraries are in ${prefix}/lib/dynamic-graph-plugin. It forces users to add this directory in the LD_LIBRARY_PATH.
Would it be relevant to scan LD_LIBRARY_PATH in https://github.com/jrl-umi3218/jrl-cmakemodules/blob/master/dynamic_graph/submodule/__init__.py.cmake and to add subdirectories `dynamic-graph-plugins` to each path?

**Comment by jmirabel** [comment]
On my computer, I do not have ${prefix}/lib/dynamic-graph-plugin in LD_LIBRARY_PATH and it works.

**Comment by florent-lamiraux** [comment]
Can you check that the linker does not find the version in your `build` directory ?

**Comment by florent-lamiraux** [comment]
modifying LD_LIBRARY_PATH using `os` python module does not help. I have just tried. I think I will add `/opt/openrobots/lib/dynamic-graph-plugins` in my `LD_LIBRARY_PATH`

**Comment by jmirabel** [comment]
`readelf -d wrap.so | grep runpath` gives
> 0x000000000000001d (RUNPATH)            Library runpath: [/home/jmirabel/devel/agimus/install/lib/dynamic-graph-plugins:/home/jmirabel/devel/agimus/src/sot-core/build-rel/src:/home/jmirabel/devel/agimus/src/sot-core/build-rel:/home/jmirabel/devel/agimus/install/lib:]

I checked and the version found is the installed version.

**Comment by florent-lamiraux** [comment]
The binary version gives
```
root@e42d29ca65b6:/opt/openrobots/lib/python2.7/site-packages/dynamic_graph/sot/core/gain_adaptive# readelf -d wrap.so | grep runpath
 0x000000000000001d (RUNPATH)            Library runpath: [/local/robotpkg/var/tmp/robotpkg/wip/py-sot-core-v3/work/sot-core-4.9.1/src:/local/robotpkg/var/tmp/robotpkg/wip/py-sot-core-v3/work/sot-core-4.9.1:/opt/openrobots/lib:]
```

**Comment by jmirabel** [comment]
@nim65s may not the solution.

**Comment by nim65s** [comment]
I proposed a solution  here: https://github.com/jrl-umi3218/jrl-cmakemodules/pull/348

**Comment by nim65s** [comment]
fixed by #61
